src/app/shared-module/nav-menu/nav-menu.component.html

```diff
@@ -21,7 +21,7 @@
   >
     <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
       <li class="nav-item active">
-        <a class="nav-link btn btn-outline-danger" href="#">Statistics</a>
+        <a class="nav-link btn btn-outline-danger" href="statistics">Statistics</a>
```

Comment:
better to use `[routerLink](https://angular.io/api/router/RouterLink)=` instead of `href`. `href` will reload(? as I remember) the page breaking all beauty of SPA :)